cashctrl_ledger/ledger.py

```diff
@@ -61,24 +58,12 @@ def restore_from_zip(self, archive_path: str):
                     f"Missing required files in the archive: {', '.join(missing_files)}"
                 )
 
-            settings = json.loads(archive.open('settings.json').read().decode('utf-8'))
-            ledger = pd.read_csv(archive.open('ledger.csv'))
-            accounts = pd.read_csv(archive.open('accounts.csv'))
-            tax_codes = pd.read_csv(archive.open('tax_codes.csv'))
+            settings = json.loads(archive.open('settings.yml').read().decode('utf-8'))
             self.restore(
                 settings=settings,
-                ledger=ledger,
-                tax_codes=tax_codes,
-                accounts=accounts,
             )
 
-    def restore(
-        self,
-        settings: dict | None = None,
-        tax_codes: pd.DataFrame | None = None,
-        accounts: pd.DataFrame | None = None,
-        ledger: pd.DataFrame | None = None,
-    ):
+    def restore(self, settings: dict | None = None):
```

Comment:
Why do we need a restore method in CashCtrlLedger?

I believe the `LedgerEngine.restore()` method will work for everything except settings.
I would implement a method `LedgerEngine.restore_settings(settings: dict)` that is invoked inside `LedgerEngine.restore()` and can be overridden in CshCtrlLedger for the specific purpose of restoring the settings.

(Btw: `restore_settings()` is equivalent to `mirror_settings()`)